tsconfig.json

```diff
@@ -12,8 +12,8 @@
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "lib",                         /* Redirect output structure to the directory. */
-    "rootDir": "./src",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    //"outDir": "lib",                         /* Redirect output structure to the directory. */
```

Comment:
moving all the path stuff to each package, but want to keep general config to extend from here